configs/FLYCOLORF7_AIO_HD/config.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        FLYCOLORF7_AIO_HD
+#define MANUFACTURER_ID   FLCO
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6500
+#define USE_BARO
+#define USE_BARO_BMP280
```

Comment:
If the baro is on SPI, this should be 
`#define USE_BARO_SPI_BMP280`

Reply:
Thanks, good catch.